src/c++/perf_analyzer/genai-pa/genai_pa/llm_metrics.py

```diff
@@ -26,12 +26,18 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import contextlib
+import io
 from dataclasses import dataclass
 from itertools import pairwise
 
 import numpy as np
 from genai_pa.utils import load_json
-from transformers import AutoTokenizer
+
+with contextlib.redirect_stdout(io.StringIO()) as stdout, contextlib.redirect_stderr(
```

Comment:
An inline comment on the purpose of this would be good

Reply:
we added on in the main.py, missed this one.
thanks for catching this

Reply:
done